### PR TITLE
Implement baritone

### PR DESCRIPTION
**Pull request by zeroeightysix**
I'd love to implement cabaletta's [baritone](https://github.com/cabaletta/baritone) into KAMI and replace the crude pathfinding Pathfind currently uses. This PR does just that!

- [x] Have baritone work and do its thing correctly
- [ ] A nice interface for users to be able to use baritone to its full extent

### CODE REVIEW

File: src/main/java/me/zeroeightsix/kami/command/commands/PathCommand.java [hunk 1]
```diff
@@ -1,52 +1,106 @@
...
-                Command.sendChatMessage("No location to retry pathfinding to.");
-                return;
-            }
-        }
-        if (args.length <= 3) {
-            Command.sendChatMessage("&cMissing arguments: x, y, z");
+        BaritoneAPI.getSettings().chatDebug.value = false;
+        BaritoneAPI.getSettings().allowSprint.value = true;
+        BaritoneAPI.getSettings().antiCheatCompatibility.value = true;
+        BaritoneAPI.getSettings().walkWhileBreaking.value = false;
```
Comment (leijurv) [inline src/main/java/me/zeroeightsix/kami/command/commands/PathCommand.java]: why?

File: src/main/java/me/zeroeightsix/kami/command/commands/PathCommand.java [hunk 2]
```diff
@@ -1,52 +1,106 @@
...
-                return;
-            }else{
-                Command.sendChatMessage("No location to retry pathfinding to.");
-                return;
-            }
-        }
-        if (args.length <= 3) {
-            Command.sendChatMessage("&cMissing arguments: x, y, z");
+        BaritoneAPI.getSettings().chatDebug.value = false;
+        BaritoneAPI.getSettings().allowSprint.value = true;
```
Comment (leijurv) [inline src/main/java/me/zeroeightsix/kami/command/commands/PathCommand.java]: This defaults true

File: src/main/java/me/zeroeightsix/kami/command/commands/PathCommand.java [hunk 3]
```diff
@@ -1,52 +1,106 @@
...
+        BaritoneAPI.getSettings().antiCheatCompatibility.value = true;
+        BaritoneAPI.getSettings().walkWhileBreaking.value = false;
+        BaritoneAPI.getSettings().colorCurrentPath.value = Color.RED;
+        BaritoneAPI.getSettings().colorNextPath.value = Color.CYAN;
+        BaritoneAPI.getSettings().colorBestPathSoFar.value = Color.YELLOW;
```
Comment (leijurv) [inline src/main/java/me/zeroeightsix/kami/command/commands/PathCommand.java]: This is disgusting but your choice LOL

**Comment by Mhowser** [comment]
any progress on this?

File: src/main/java/me/zeroeightsix/kami/command/commands/PathCommand.java (hunk 1 above)
Comment (leijurv) [reply src/main/java/me/zeroeightsix/kami/command/commands/PathCommand.java]: specificallllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllly walkwsonateuhstnaoeuh,ayp;oequi hbntmsnw-

**Comment by leijurv** [comment]
lol

**Comment by zeroeightysix** [comment]
@Mhowser I'm thinking of axing this and encouraging users to use the standalone mod instead (though optional integration might come later)

**Comment by Mhowser** [comment]
ah ok. Is making the GUI the current roadblock?

**Comment by leijurv** [comment]
pls integrate with separate api jar uwu

**Comment by ChristoferMinestar** [comment]
@leijurv @zeroeightysix how about we support baritone as a plugin

**Comment by zeroeightysix** [comment]
Closing: inactivity

**Pull request closed without merging**